src/client/common/application/commands.ts

```diff
@@ -58,6 +59,7 @@ interface ICommandNameWithoutArgumentTypeMapping {
     [DSCommands.CreateNewNotebook]: [];
     [DSCommands.EnableDebugLogging]: [];
     [DSCommands.ResetLoggingLevel]: [];
+    [DSCommands.OpenVariableView]: [];
```

Comment:
@rchiodo I like the idea of hooking this up to a button on the native notebook toolbar. @claudiaregio thoughts on this? I think I just need an icon to do that.